README.md

```diff
@@ -214,6 +197,29 @@ You're also more than welcome to:
 - comment an issue, discussing the details
 - open a PR, fixing a given issue
 
+### Setting up local development which means running the docs/demo locally:
```

Comment:
Why you moved local set up from installation to contributing? I don't understand this decision.

Reply:
only contributors need to set up docs locally and run tests;

people, who want only use carousel, can click the deployed docs and see the tests result in CI

Reply:
ok you convinced me